### ISSUE TITLE
WCAG Compliance 4.1.2 Name, Role, Value

### ISSUE THREAD

**Issue by EzraSkwarka** [RunestoneInteractive/RunestoneComponents#958]
We are verifying and improving Runestone's level of WCAG compliance to Level AA, our findings are summarized on a Google Doc [here](https://docs.google.com/document/d/1XKiq4WanGln44by62XkSvAqjpse_mElYNoYtCIvtgBk/edit?usp=sharing). _This is for guideline 4.1.2_

>**4.1.2 Name, Role, Value**: For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies. (Level A)

The only issue we could find with this topic is that for the arrows between chapters we need to list where they go _before_ they are hovered over.

Part of the Berea College Open Source Class @clarkh97 @owenslo @EzraSkwarka

**Comment by EzraSkwarka** [comment]
The arrows also violate 1.3.3 
>**1.3.3 Sensory Characteristics:** Instructions provided for understanding and operating content do not rely solely on sensory characteristics of components such as shape, size, visual location, orientation, or sound. (Level A)
Note: For requirements related to color, refer to Guideline 1.4.
